hsmodels/schemas/__init__.py

```diff
@@ -84,7 +85,7 @@ def parse_file(schema, file, file_format='xml', subject=None):
 def rdf_graph(schema):
     for rdf_schema, user_schema in user_schemas.items():
         if isinstance(schema, user_schema):
-            return _rdf_graph(rdf_schema(**schema.dict(to_rdf=True)), Graph())
+            return _rdf_graph(rdf_schema(**schema.model_dump(to_rdf=True)), Graph())
```

Comment:
Wh is exclude_none=True included in line 75 but not here?

Reply:
For the RDF schema models we are using the `model_dump() `provided by pydantic `BaseModel `in which the default is `exclude_none=Fasle`. For the metadata schemas we are using our `BaseMetadata `class where we have a customized `model_dump() `using default as` exclude_none=True`.